constants.js

```diff
@@ -179,6 +179,18 @@ const constants = {
     assumedRoleArnResourceType: 'assumed-role',
     // Session name of the backbeat lifecycle assumed role session.
     backbeatLifecycleSessionName: 'backbeat-lifecycle',
+    possibleSignatureChecksums: new Set([
```

Comment:
To avoid duplicates you could turn it into `unsupportedSignatureChecksums` and remove the supported ones, I think the behavior would stay the same.